### DISCUSSION TITLE
[FEATURE-REQUEST] Cómo se puede ocultar el cambio de estado de los 3 puntos de una tasks?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#172] [3. Ideas & Feature Requests] [answered]
Un cliente me preguntó que si se puede quitar los 3 puntos la opción de cambiar estados, creo que si el user es un editor lo puede hacer, pero me gustaría confirmar dicha información.

Adjunto imagen
![image](https://user-images.githubusercontent.com/61419471/117891449-ead22580-b284-11eb-991d-82245a291fcb.png)

**Comment by rafaelwalter** [comment]
@gmoplaza podrias contarnos un poco sobre la nueva funcionalidad que estás trabajando para hacer cambios de estado con 'Actions'? reemplazaria esto que comenta @NicolasPLG?

**Comment by acedward** [comment, marked as answer]
Por el momento no es posible. Pero viene un cambio dentro de el/los próximo(s) mes(es) donde se podrá (no tenemos fecha exacta de release)

Si necesitas priorizar esta solicitud por favor habla con tu representante Cotalker

**Reply by NicolasPLG** [reply]
gracias @acedward por la información.

Atte,
